### ISSUE TITLE
test issue to see if twisted-bugs ML is working.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10362]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#472|
|Type|enhancement|
|Created|2004-01-13 13:43:50Z|

<details><summary>Searchable metadata</summary>

```
trac-id__472 472
type__enhancement enhancement
reporter__radix radix
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1074001430000000 1074001430000000
changetime__1470656475226602 1470656475226602
version__ 
owner__ 
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
HOOPDY DOO.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
And to see if modifications are also sent...
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
eh... let's only send it for new issues.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
Ok! It works. Off to close the real issue about this.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|GitHub commented|
|-|-|

In changeset d3bb36fa54cecd84404f53ea375df9843cf6d90c
```
#!CommitTicketReference repository="" revision="d3bb36fa54cecd84404f53ea375df9843cf6d90c"
Merge pull request #472 from twisted/8718-rodrigc-stateful-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8718

Fix twisted.test.test_stateful on Python 3
```
